manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Shemnei.punktf
+PackageVersion: 3.0.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: punktf-x86_64-pc-windows-msvc/punktf.exe
+  PortableCommandAlias: punktf
+ReleaseDate: 2022-12-23
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
+- Architecture: x86
+  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56

```

Comment:
```suggestion
  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: x86
  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/2ef8a2c4-a974-41f3-9256-3da4a47d26e5)

Reply:
Thanks!

Reply:
Thanks! Are these kind of dependencies yet supported, or would we have to wait for https://github.com/microsoft/winget-cli/issues/163? 
The author just released a static version which does not depend on this runtime. If these kind of dependencies are not yet supported, I would close this PR and create a new one which uses the static dependencies.